frame/treasury/src/lib.rs

```diff
@@ -453,9 +458,29 @@ pub mod pallet {
 			beneficiary: AccountIdLookupOf<T>,
 		) -> DispatchResult {
 			let max_amount = T::SpendOrigin::ensure_origin(origin)?;
-			let beneficiary = T::Lookup::lookup(beneficiary)?;
-
 			ensure!(amount <= max_amount, Error::<T, I>::InsufficientPermission);
+
```

Comment:
This change is carried over from a cherry picked commit and won't be in the final PR. Please ignore.